packages/apollo-forest-run/src/forest/types.ts

```diff
@@ -57,6 +60,18 @@ export type IndexedForest = {
 export type Source = Readonly<SourceObject | SourceCompositeList>;
 export type Draft = SourceObject | SourceCompositeList;
 
+export type UpdateChunkMetadata = {
+  type: string;
+  depth: number;
+  siblings: number;
+  dirtyFields: Set<string>;
+};
+
+export type UpdateForestMetadata = {
+  totalUpdated: number;
```

Comment:
This will likely be the same as `operationCosts.size`? So maybe just return the map with update stats for now?